settings/installer/ospd/ospd.spec

```diff
@@ -53,6 +53,13 @@ subparsers:
                     help: The overcloud storage type
                     default: no-storage.yml
 
+            - title: Swift object-storage
```

Comment:
no need to make this a dedicated flag, use teh "storage" flag and make a "swift.yml" option there